packages/forms/src/styled/file-upload/StyledFileUpload.ts

```diff
@@ -77,11 +61,41 @@ const colorStyles = (props: ThemeProps<DefaultTheme> & IStyledFileUploadProps) =
   `;
 };
 
+const sizeStyles = (props: ThemeProps<DefaultTheme> & IStyledFileUploadProps) => {
+  const marginTop = props.theme.space.base * (props.isCompact ? 1 : 2);
+  const paddingHorizontal = props.theme.space.base * (props.isCompact ? 7.5 : 15);
+  const paddingVertical = math(
+    `${props.theme.space.base * (props.isCompact ? 2.5 : 5)} - ${props.theme.borderWidths.sm}`
+  );
+  const fontSize = props.theme.fontSizes.md;
+  const lineHeight = getLineHeight(props.theme.space.base * 5, fontSize);
+
+  return css`
+    padding: ${paddingVertical} ${paddingHorizontal}px;
```

Comment:
Is `paddingVertical` missing a unit (eg `px`) here?

Reply:
`paddingVertical` is actually returning the value with pixels. 😳
![image](https://user-images.githubusercontent.com/30312964/142683258-2541ea68-d543-46c5-821d-61e45c539568.png)

Reply:
I see now, looks like [polished.math](https://polished.js.org/docs/#math) takes care of that!

Reply:
Mixing the declared `const` formats wasn't great for readability. I also realized that there was a missed opportunity to express font-relative spacing (specifically, horizontal padding) in `em` units. Updated.